scene/animation/animation_player.cpp

```diff
@@ -748,7 +748,7 @@ void AnimationPlayer::_animation_process_animation(AnimationData *p_anim, double
 				if (!nc->node) {
 					continue;
 				}
-				if (!p_is_current) {
+				if (!p_is_current || is_stopping) {
```

Comment:
Nit pick for consistency:
```cpp
			case Animation::TYPE_METHOD: {
				if (!nc->node || is_stopping) {
					continue;
				}
```
However, it is doubtful what this brake exists for, so it may be done as follows.
```cpp
			case Animation::TYPE_METHOD: {
				if (!nc->node || is_stopping || !p_is_current) {
					continue;
				}
```